lib/vauth/ntlm_sspi.c

```diff
@@ -262,11 +262,37 @@ CURLcode Curl_auth_create_ntlm_type3_message(struct Curl_easy *data,
 
   /* Setup the type-2 "input" security buffer */
   type_2_desc.ulVersion = SECBUFFER_VERSION;
-  type_2_desc.cBuffers  = 1;
-  type_2_desc.pBuffers  = &type_2_buf;
-  type_2_buf.BufferType = SECBUFFER_TOKEN;
-  type_2_buf.pvBuffer   = ntlm->input_token;
-  type_2_buf.cbBuffer   = curlx_uztoul(ntlm->input_token_len);
+  type_2_desc.cBuffers = 1;
+  type_2_desc.pBuffers = &type_2_bufs[0];
+
+
```

Comment:
Can I suggest we just remove these extra newlines?

Reply:
Happy to do it. Do you prefer a separate commit or amend the existing?